arch/arm/core/cortex_m/mpu/arm_mpu.c

```diff
@@ -109,18 +109,18 @@ static inline u32_t _get_region_attr_by_type(u32_t type, u32_t size)
 	switch (type) {
 #ifdef CONFIG_USERSPACE
 	case THREAD_STACK_REGION:
-		return _get_region_attr(1, P_RW_U_RW, 0, 1, 0,
-					1, 0, region_size);
+		return _get_region_attr(1, P_RW_U_RW, 1, 1, 1,
+					0, 0, region_size);
 #endif
 #ifdef CONFIG_MPU_STACK_GUARD
 	case THREAD_STACK_GUARD_REGION:
-		return _get_region_attr(1, P_RO_U_NA, 0, 1, 0,
-					1, 0, region_size);
+		return _get_region_attr(1, P_RO_U_NA, 1, 1, 1,
+					0, 0, region_size);
 #endif
 #ifdef CONFIG_APPLICATION_MEMORY
 	case THREAD_APP_DATA_REGION:
-		return _get_region_attr(1, P_RW_U_RW, 0, 1, 0,
-					1, 0, region_size);
+		return _get_region_attr(1, P_RW_U_RW, 1, 1, 1,
```

Comment:
@aurel32: Here shareable attribute is removed from the region.

Reply:
Oh indeed, that's a mistake, that's why it is not mentioned in the commit message. I'll fix that.